### PR TITLE
Update electra types for devnet1 compatibility

### PR DESCRIPTION
**Pull request by pk910**
Updated new electra types to latest version.

### CODE REVIEW

File: http/http.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (s *Service) post(ctx context.Context, endpoint string, body io.Reader) (io
 		return nil, errors.Join(errors.New("failed to read POST response"), err)
 	}
 
-	statusFamily := resp.StatusCode / 100
```
Comment (mcdee) [inline http/http.go:82]: Please can you revert these changes?  They have been addressed in master.

File: http/http.go [hunk 2]
```diff
@@ -223,7 +223,7 @@ func (s *Service) post2(ctx context.Context,
 		}
 	}
 
-	statusFamily := resp.StatusCode / 100
```
Comment (mcdee) [inline http/http.go:226]: Please can you revert these changes? They have been addressed in master.

File: http/http.go [hunk 3]
```diff
@@ -381,7 +381,7 @@ func (s *Service) get(ctx context.Context,
...
```
Comment (mcdee) [inline http/http.go:384]: Please can you revert these changes? They have been addressed in master.

File: multi/client.go [hunk 4]
```diff
@@ -170,7 +171,7 @@ func (s *Service) doCall(ctx context.Context, call callFunc, errHandler errHandl
 			log.Trace().Err(err).Msg("Potentially deactivating client due to error")
 			var apiErr *api.Error
 			switch {
-			case errors.As(err, &apiErr) && apiErr.StatusCode/100 == 4:
```
Comment (mcdee) [inline multi/client.go:173]: Please can you revert these changes? They have been addressed in master.

File: spec/electra/executionpayload_json.go [hunk 5]
```diff
@@ -30,25 +30,26 @@ import (
...
+	GasUsed               string                     `json:"gas_used"`
+	Timestamp             string                     `json:"timestamp"`
+	ExtraData             string                     `json:"extra_data"`
+	BaseFeePerGas         string                     `json:"base_fee_per_gas"`
+	BlockHash             phase0.Hash32              `json:"block_hash"`
+	Transactions          []string                   `json:"transactions"`
+	Withdrawals           []*capella.Withdrawal      `json:"withdrawals"`
+	BlobGasUsed           string                     `json:"blob_gas_used"`
+	ExcessBlobGas         string                     `json:"excess_blob_gas"`
+	DepositRequests       []*DepositRequest          `json:"deposit_receipts"`
```
Comment (mcdee) [inline spec/electra/executionpayload_json.go:50]: The JSON tag should be `deposit_requests`, no?

File: spec/electra/executionpayload_yaml.go [hunk 6]
```diff
@@ -25,25 +25,26 @@ import (
...
+	GasUsed               uint64                  `yaml:"gas_used"`
+	Timestamp             uint64                  `yaml:"timestamp"`
+	ExtraData             string                  `yaml:"extra_data"`
+	BaseFeePerGas         string                  `yaml:"base_fee_per_gas"`
+	BlockHash             string                  `yaml:"block_hash"`
+	Transactions          []string                `yaml:"transactions"`
+	Withdrawals           []*capella.Withdrawal   `yaml:"withdrawals"`
+	BlobGasUsed           uint64                  `yaml:"blob_gas_used"`
+	ExcessBlobGas         uint64                  `yaml:"excess_blob_gas"`
+	DepositRequests       []*DepositRequest       `yaml:"deposit_receipts"`
```
Comment (mcdee) [inline spec/electra/executionpayload_yaml.go:45]: Again, should be `deposit_requests`?

**Review by mcdee — CHANGES_REQUESTED** [review]
A few comments/changes requested.  I'd also rebase this against `master` as that is more up-to-date with the changes required for the latest version of golangci-lint.

File: spec/electra/executionpayload_json.go (hunk 5 above)
Comment (pk910) [reply spec/electra/executionpayload_json.go:50]: good catch :)

File: spec/electra/executionpayload_yaml.go (hunk 6 above)
Comment (pk910) [reply spec/electra/executionpayload_yaml.go]: fixed & found even more places where it was still called `deposit_receipts`

**Review by mcdee — APPROVED** [review]
